apps/expo/src/components/spotify/Spotify.tsx

```diff
@@ -0,0 +1,51 @@
+import { Dimensions, Image, Text, View } from 'react-native'
+import { StyleSheet } from 'react-native'
+
+import { GlobalContext } from '~/context/global-context'
+import { api } from '~/utils/api'
+
+// Use to show the progress of the song using a bar
+import SongProgress from 'apps/expo/src/components/spotify/song_progress'
+
+export default function Spotify({inputID}: {inputID: number}) {
+    // Should use this line instead for actual display
+    const {data} = api.spotify.getSpotifyDataFromUserId.useQuery({id: inputID})
+
+    if (data == null || data == undefined || data.isPlaying != true) {
+    return (
+        <View>
+        <Text>User is not actively listening to Spotify.</Text>
+        </View>
+    )
+    }
+
+    let albumImageURL: string = data.albumImageURL ? data.albumImageURL : ''
+    let songTitle: string = data.title ? data.title : ''
+    let currTime: number = data.timePlayed? data.timePlayed : 0
+    let totalTime: number = data.timeTotal? data.timeTotal : 0
+    let artists: string = data.artist? data.artist : ''
+    let album: string = data.albumName? data.albumName : ''
+
+    // Make a progress bar to display it: SongProgress handles time and progress bar
+    // Should make a rotating weight plate for aesthetic reasons?
+
+    return (
```

Comment:
Good markdown formatting here. brownie points for you

Reply:
my commit will accomodates the format🤣